Resources/Prototypes/Entities/Clothing/Masks/specific.yml

```diff
@@ -24,6 +24,41 @@
       slots:
       - Snout
 
+- type: entity
+  parent: ClothingMaskBase
+  id: ClothingMaskGasChameleonArmored
+  name: gas mask
+  description: A face-covering mask that can be connected to an air supply.
+  suffix: Armored, Chameleon
+  components:
+    - type: Tag
+      tags: [] # ignore "WhitelistChameleon" tag
+    - type: Sprite
+      sprite: Clothing/Mask/gas.rsi
+    - type: Clothing
+      sprite: Clothing/Mask/gas.rsi
+    - type: ChameleonClothing
+      slot: [mask]
+      default: ClothingMaskGas
+    - type: BreathMask
+    - type: IdentityBlocker # need that for default ClothingMaskGas
+    - type: UserInterface
+      interfaces:
+        enum.ChameleonUiKey.Key:
+          type: ChameleonBoundUserInterface
+    - type: HideLayerClothing
+      slots:
+      - Snout
+    - type: FlashImmunity
+    - type: EyeProtection
+    - type: Armor
+      modifiers:
+        coefficients:
+          Blunt: 0.95
+          Slash: 0.95
+          Piercing: 0.95
+          Heat: 0.95
```

Comment:
Does this even really do anything? I guess it saves you from crit from a few seconds if you get shot with exactly 100 pierce damage... but you'll fall to bleeding anyway.

Reply:
Something is better than nothing, and that's not the main appeal of the mask. The main appeal lies in the Eye Protection, the fact that armor applies multiplicatively means it does barely anything.